graphql/resolvers/exerciseCrud.ts

```diff
@@ -102,6 +104,41 @@ export const deleteExercise = withUserContainer<
   })
 })
 
+// replaces deleteExercise
+export const removeExercise = withUserContainer<
+  Promise<Exercise>,
+  MutationRemoveExerciseArgs
+>(async (_parent: void, args: MutationRemoveExerciseArgs, ctx: Context) => {
+  const { req } = ctx
+  const { id } = args
+
+  const exercise = await prisma.exercise.findUnique({
+    where: {
+      id
+    }
+  })
+
+  const authorId = req.user?.id
+
+  if (!isAdmin(req) && exercise?.authorId !== authorId)
```

Comment:
This if has no curly braces, the next one does, both have single line statements. This is just a style suggestion but can you make them uniform (preferably add braces to both)